src/Saved/Saved.css

```diff
@@ -0,0 +1,8 @@
+span {
+  margin-bottom: 10px;
+}
```

Comment:
interesting use of an additional span rule to only apply to your saved view. Hadnt thought of doing that and may end up doing something similar